CRM/Utils/Cache/APCcache.php

```diff
@@ -97,10 +101,12 @@ public function set($key, $value, $ttl = NULL) {
    * @return mixed
    */
   public function get($key, $default = NULL) {
-    if ($default !== NULL) {
-      throw new \RuntimeException("FIXME: " . __CLASS__ . "::get() only supports NULL default");
+    CRM_Utils_Cache::assertValidKey($key);
+    $result = apc_fetch($this->_prefix . $key, $success);
```

Comment:
$success does not seem to be defined

Reply:
That sounds like some metaphysical management widsom. ;)

More seriously, `$success` is actually an output of this line. [apc_fetch($key, &$success)](http://php.net/apc_fetch) sets the value of `$success` to indicate whether it worked.